style.css

```diff
@@ -117,15 +119,268 @@ a {
   border-top: 34px;
   height: 20px;
   padding: 0;
+  margin-bottom: 62px;
 }
 
 .socials li {
   padding: 10px 10px 10px 10px;
 }
 
+.recent-works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0 24px 0 24px;
+}
+
 .arrow {
-  display: block;
-  margin-top: 60px;
   height: 40px;
+  width: 40px;
+  text-align: center;
+}
+
+.title {
+  margin-right: 26px;
+  margin-left: 26px;
+}
+
+.title p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  height: 97px;
+  text-align: center;
+  color: #172b4d;
+  margin-top: 22px;
+  margin-bottom: 24px;
+}
+
+.bar {
+  background: url("assets/images/Indicator.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+  border-radius: 24px;
+  text-align: center;
+  height: 4px;
+  width: 48px;
+}
+
+.recent-works-cards {
+  margin: 58px 0 0 0;
+  padding: 4px 2px;
+  height: 2972px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.works {
+  background: url("assets/images/Img-Plaholder.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 474px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.work-details {
+  display: grid;
+  grid-template-rows: 33.33% 33.33% 33.33%;
+  height: 222px;
+  background: #fff;
+  border-radius: 8px;
+  align-content: space-between;
+  align-items: center;
+  justify-items: center;
+}
+
+.work-details p {
+  margin: 12px 0 0 0 !important;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  text-align: center;
+  color: #3a4a42;
+  padding: 0 10px;
+}
+
+.work-details ul {
+  margin: 0;
+}
+
+.language-tags {
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  justify-content: space-evenly;
+  width: 100%;
+  padding: 0;
+}
+
+.language-tags li {
+  list-style: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+  padding: 8px 8px;
+  background: #ebf0ee;
+  border-radius: 4px;
+}
+
+.works-button {
+  padding: 12px;
+  width: 98px;
```

Comment:
- [ ] Please, Add the button styling (Hover, enable, Active, Pressed ) 
Figma design :
![image](https://user-images.githubusercontent.com/71874425/153665008-cc215d60-de79-47fa-9531-d40c0b6ce644.png)